src/renderer/src/routes/Onboarding/index.tsx

```diff
@@ -1,36 +1,46 @@
-import { styled } from '@mui/material/styles'
-import { createFileRoute, useNavigate } from '@tanstack/react-router'
+import Box from '@mui/material/Box'
+import Container from '@mui/material/Container'
+import Grid from '@mui/material/Grid'
+import List from '@mui/material/List'
+import ListItem from '@mui/material/ListItem'
+import ListItemText from '@mui/material/ListItemText'
+import Stack from '@mui/material/Stack'
+import Typography from '@mui/material/Typography'
+import { alpha, useTheme } from '@mui/material/styles'
+import useMediaQuery from '@mui/material/useMediaQuery'
+import { createFileRoute } from '@tanstack/react-router'
 import { defineMessages, useIntl } from 'react-intl'
 
-import { CORNFLOWER_BLUE, DARK_COMAPEO_BLUE, ORANGE, WHITE } from '../../colors'
-import { Button } from '../../components/Button'
-import { Text } from '../../components/Text'
-import TopoBackground from '../../images/TopoLogo.svg'
-import Calling from '../../images/calling.png'
-import LockedWithKey from '../../images/closed_lock_with_key.png'
-import RaisedFistMediumSkinTone from '../../images/raised_fist_medium_skin_tone.png'
-import WorldMap from '../../images/world_map.png'
+import { BLACK, BLUE_GREY, DARK_COMAPEO_BLUE } from '../../colors'
+import { ButtonLink } from '../../components/button-link'
+// TODO: Remove `?url` after getting rid of vite-plugin-svgr
+import TOPO_IMAGE_URL from '../../images/TopoLogo.svg?url'
```

Comment:
highlighting for future reference. without the `?url`, this import gets processed by the svgr plugin and the import becomes a react component, which is not desired in this case especially.